src/cljs/rems/dropdown.cljs

```diff
@@ -0,0 +1,52 @@
+(ns rems.dropdown
+  (:require [cljsjs.react-select]
+            [rems.guide-functions]
+            [rems.text :refer [text]])
+  (:require-macros [rems.guide-macros :refer [component-info example]]))
+
+;; TODO: add an ID to the input field and then link all labels to it
+(defn dropdown
+  "Single- or multi-choice, searchable dropdown menu."
+  [{:keys [items item->label item->selected? item->disabled? multi? on-change]
+    :or {item->selected? (constantly false)
+         item->disabled? (constantly false)}}]
+  (let [options (map (fn [item] {:value item
+                                 :label (item->label item)
+                                 :isDisabled (item->disabled? item)})
+                     items)
+        grouped (group-by #(item->selected? (% :value)) options)]
+    [:> js/Select {:className "dropdown-container"
+                   :classNamePrefix "dropdown-select"
+                   :isMulti multi?
+                   :maxMenuHeight 200
+                   :noOptionsMessage #(text :t.dropdown/no-results)
+                   :options (if multi?
+                              (or (get grouped false) [])
```

Comment:
aka `(get grouped false [])`

Reply:
Done. I had completely forgotten about this syntax...